File: src/LPII/Atividade1/Exercicio6.java
package LPII.Atividade1;

import java.util.Scanner;

public class Exercicio6 {
    public static void main(String[] args) {
        Scanner ler = new Scanner(System.in);

        int[] vetorx= new int[10];
        int[] vetory= new int[10];
        int i, j, z;

        for(i=0; i<10; i++) {
            System.out.println("Digite os numeros do vetor de X");
            vetorx[i] = ler.nextInt();
        }
        for(j=0; j<10; j++) {
            System.out.println("Digite os numeros do vetor de Y");
            vetory[j] = ler.nextInt();
        }

        System.out.println("\nUniao de X e Y");
        for(i=0; i<10; i++){
            System.out.println(vetorx[i]);
        }
        for(j=0; j<10; j++){
            System.out.println(vetory[j]);
        }

        System.out.println("\nDiferenca X - Y");
        for(i=0; i<10; i++){
            System.out.println(vetorx[i]-vetory[i]);
        }

        System.out.println("\nSoma de X + Y");
        for(i=0; i<10; i++){
            System.out.println(vetorx[i]+vetory[i]);
        }

        System.out.println("\nMultiplicacao de X * Y");
        for(i=0; i<10; i++){
            System.out.println(vetorx[i]*vetory[i]);
        }

        System.out.println("\nInterseccao de X e Y");
        for(i=0; i<10; i++){
            z =vetorx[i];
            for(j=0; j<10; j++){
                if(z == vetory[j]){
                    System.out.println(z);
                    break;
                }
            }
        }
    }







}

File: src/LPII/Atividade1/Exercicio6.c
package LPII.Atividade1;

public class Exercicio6 {
    #include <stdio.h>
    #define v 10

    int main(){

        int vetorx[v], vetory[v], pu[20], pd[v], ps[v], pp[v], pi[v], cont, x, y;

        for(cont=0; cont<v; cont++){
            printf("Digite o %d numero do vetor X\n", cont+1);
            scanf("%d", &vetorx[cont]);
        }
        for(cont=0; cont<v; cont++){
            printf("Digite o %d numero do vetor Y\n", cont+1);
            scanf("%d", &vetory[cont]);
        }
        for(cont=0; cont<v; cont++) {
            pu[cont*2] = vetorx[cont];
        }
        for(cont=0; cont<v; cont++) {
            pu[cont*2+1] = vetory[cont];
        }
        printf("\n\na)Uniao de X e Y:\n");
        for(cont=0; cont<20; cont++) {
            printf("%d ", pu[cont]);
        }
        printf("\n\n");
        for(cont=0; cont<v; cont++){
            pd[cont]=vetorx[cont]-vetory[cont];
            printf("\tb) X - Y = %d\n", pd[cont]);
        }
        printf("\n\n");
        for(cont=0; cont<v; cont++){
            ps[cont]=vetorx[cont]+vetory[cont];
            printf("c) X + Y = %d\n", ps[cont]);
        }
        printf("\n\n");
        for(cont=0; cont<v; cont++){
            pp[cont]=vetorx[cont]*vetory[cont];
            printf("\td) X * Y = %d\n", pp[cont]);
        }

        printf("\n\n");
        printf("\ne)Interseccao de X e Y:\n");
        for(x=0; x<v; x++){
            for(y=0; y<v; y++){
                if(vetorx[x]==vetory[y]){
                    printf("%d ", vetorx[x] );
                }
            }
        }
        printf("\n\n");

        return 0;
    }
}
